### DISCUSSION TITLE
Guides on how to make models

### DISCUSSION THREAD

**Discussion by SolaireRocks** [civitai/civitai#214] [Ideas]
I'm having a heck of a time getting training down right and it would be nice if you had guides on how to make great LoRAs, checkpoints, textual inversions, etc on this site. The more people who know how to do it, the more people who can contribute to the site.

**Comment by JustMaier** [comment]
We've got a [Questions section](https://civitai.com/questions) of the site that we were hoping to have be a home for questions like "How do I train a LoRA?" but we've been pulled hard in other directions since shortly after we launched the beta of that section of the site.

We'd also like to put together a Community Articles section at some point which I imagine could also be a great home for content that answers questions like this.

Either way, if someone hasn't already asked the question about training there, maybe ask it and we can start getting some decent answers :)
